launch_testing/launch_testing_examples/launch_testing_examples/check_msgs_launch_test.py

```diff
@@ -14,14 +14,14 @@
 
 import unittest
 
+from example_interfaces.msg import String
```

Comment:
i believe this changes led to https://ci.ros2.org/job/ci_linux/20965/testReport/junit/launch_testing_examples.launch_testing_examples/check_msgs_launch_test/launch_testing_examples_check_msgs_launch_test/.

this is because message type does not match anymore,

`demo_nodes_cpp` uses `std_msgs/String`, so i think this change requires the same change to https://github.com/ros2/demos